packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/TabPanels/PaymentTabPanel/UpdatePDF/UpdatePDFSection.tsx

```diff
@@ -0,0 +1,46 @@
+import { components } from '@/api';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import { Box, Typography } from '@mui/material';
+
+import { UpdatePDFModal } from './UpdatePDFModal';
+
+type Props = {
+  invoice: components['schemas']['InvoiceResponsePayload'];
+};
+
+export const UpdatePDFSection: React.FC<Props> = ({ invoice }) => {
```

Comment:
We don't use here React.FC, we pass props not as generics but as like this 
`export const UpdatePDFSection= ({ invoice }: Props)`